packages/gatsby/src/redux/actions.js

```diff
@@ -48,7 +49,7 @@ const pascalCase = _.flow(_.camelCase, _.upperFirst)
  * })
```

Comment:
It doesn't let me comment above, but the following seen above doesn't have matching quotes:
component: path.resolve('./src/templates/my-sweet-new-page.js`),